op-batcher/batcher/driver.go

```diff
@@ -158,22 +158,31 @@ func (l *BatchSubmitter) StartBatchSubmitting() error {
 	}
 
 	receiptsCh := make(chan txmgr.TxReceipt[txRef])
-	receiptsLoopCtx, cancelReceiptsLoopCtx := context.WithCancel(context.Background())
-	throttlingLoopCtx, cancelThrottlingLoopCtx := context.WithCancel(context.Background())
+
+	l.txpoolMutex.Lock()
+	l.txpoolState = TxpoolGood
+	l.txpoolMutex.Unlock()
```

Comment:
Do we even need to lock the mutex here? I think it's setup code and the other routines read/writing it are started later.

Reply:
488352e57bee620a535654cbb3b38b50019b2286